examples/ecs/query_bundle.rs

```diff
@@ -0,0 +1,69 @@
+use bevy::{
+    // diagnostic::LogDiagnosticsPlugin,
+    ecs::schedule::RunOnce,
+    log::{LogPlugin, LogSettings},
+    prelude::*,
+};
+
+fn main() {
+    App::build()
+        .insert_resource(LogSettings {
```

Comment:
To keep this simple, we should probably just remove this resource and change the log level used in the example systems to `info!`.